src/bridging-script/ios-bridge.js

```diff
@@ -0,0 +1,29 @@
+let MiniAppBridge = {}
+var queue = []
+
+MiniAppBridge.exec = function(action, onSuccess, onError) {
+    const callback = {}
+    callback.onSuccess = onSuccess;
+    callback.onError = onError;
+    callback.id = Math.random();
```

Comment:
Could we just use a sequential ID here? i.e. something like `callback.id = MiniAppBridge.callbackId++`

Reply:
In that case, we would be needing an extra global variable for this. Is that ok?